tasks/prelim.yml

```diff
@@ -249,26 +249,20 @@
         yum:
             name: aide
             state: present
+        notify: init aide
 
       - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Check for existing AIDE database"
         stat:
           path: "{{ rhel7stig_aide_db_file }}"
         register: rhel7stig_aide_db_status
         check_mode: no
+        changed_when: not rhel7stig_aide_db_status.stat.exists
+        notify: init aide
 
-      - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Initialize AIDE"
-        command: /usr/sbin/aide --init -B 'database_out=file:{{ rhel7stig_aide_db_file }}'
-        async: "{{ rhel7stig_aide_init_timeout | default(omit) }}"
-        poll: "{{ (rhel7stig_aide_init_timeout is defined) | ternary(0,omit) }}"
-        register: rhel7stig_aide_init_status
-        when: not rhel7stig_aide_db_status.stat.exists
-
-      - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Notify AIDE init handler"
-        command: "true"
-        when:
-          - rhel7stig_aide_init_status.changed
-          - rhel7stig_wait_for_aide_init
-        notify: init aide status
+      - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Set AIDE database output file"
```

Comment:
There's no need for this task.  Always write to the temp file.  Worst case, put it in defaults or vars.